graylog2-web-interface/src/views/components/sidebar/FieldList.jsx

```diff
@@ -4,7 +4,7 @@ import createReactClass from 'create-react-class';
 import PropTypes from 'prop-types';
 import { is } from 'immutable';
 import { isEqual } from 'lodash';
-import { FixedSizeList as List } from 'react-window';
```

Comment:
Why is the usage of `react-window` removed? It is not used to auto-size the field list, but to virtualize it, so only the items currently visible are rendered (like `react-virtualized`).